src/plugins/intel_gpu/tests/unit/transformations/convert_fc_to_compressed_test.cpp

```diff
@@ -463,6 +463,147 @@ TEST_F(TransformationTestsF, ConvertFCToCompressed10) {
     }
 }
 
+TEST_F(TransformationTestsF, ConvertFCToCompressed11) {
```

Comment:
[random spot] Please add functional accuracy tests
It looks like you can extend an existing test with additional parameters
src/plugins/intel_gpu/tests/functional/subgraph_tests/dynamic/matmul_weights_decompression.cpp